### ISSUE TITLE
Python example server logs "No handlers" if exception raised in method handler 

### ISSUE THREAD

**Issue by lidizheng** [grpc/grpc#17321]
### What version of gRPC and what language are you using?
```
Name: grpcio
Version: 1.16.1
Summary: HTTP/2-based RPC framework
Home-page: https://grpc.io
Author: The gRPC Authors
Author-email: grpc-io@googlegroups.com
License: Apache License 2.0
```
 
### What operating system (Linux, Windows, …) and version?
```
macOS 10.13.6
```
 
### What runtime / compiler are you using (e.g. python version or version of gcc)
Python 2.7 / 3.7
 
### What did you do?
```Python
from concurrent import futures
import time

import grpc

import helloworld_pb2
import helloworld_pb2_grpc

_ONE_DAY_IN_SECONDS = 60 * 60 * 24


class Greeter(helloworld_pb2_grpc.GreeterServicer):

    def SayHello(self, request, context):
        raise Exception('Log me please')
        return helloworld_pb2.HelloReply(message='Hello, %s!' % request.name)


def serve():
    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
    helloworld_pb2_grpc.add_GreeterServicer_to_server(Greeter(), server)
    server.add_insecure_port('[::]:50051')
    server.start()
    try:
        while True:
            time.sleep(_ONE_DAY_IN_SECONDS)
    except KeyboardInterrupt:
        server.stop(0)


if __name__ == '__main__':
    serve()
```
 
### What did you expect to see?
Output stack trace or no output
```
ERROR:grpc._server:Exception calling application: Log me please
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/grpc/_server.py", line 389, in _call_behavior
    return behavior(argument, context), True
  File "greeter_server.py", line 31, in SayHello
    raise Exception('Log me please')
Exception: Log me please
```
 
### What did you see instead?
```
No handlers could be found for logger "grpc._server"
```

### Anything else we should know about your project / environment?
N/A

### Fix
This is a long-lasting problem with Python logging. The simplest fix would be calling `logging.basicConfig()` before serving. User may start from copying the example, I think we should add it to the example script to avoid future confusion.

```Python
from concurrent import futures
import time
import logging # <- THIS

import grpc

import helloworld_pb2
import helloworld_pb2_grpc

_ONE_DAY_IN_SECONDS = 60 * 60 * 24


class Greeter(helloworld_pb2_grpc.GreeterServicer):

    def SayHello(self, request, context):
        return helloworld_pb2.HelloReply(message='Hello, %s!' % request.name)


def serve():
    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
    helloworld_pb2_grpc.add_GreeterServicer_to_server(Greeter(), server)
    server.add_insecure_port('[::]:50051')
    server.start()
    try:
        while True:
            time.sleep(_ONE_DAY_IN_SECONDS)
    except KeyboardInterrupt:
        server.stop(0)


if __name__ == '__main__':
    logging.basicConfig() # <- THIS
    serve()
```

**Comment by srini100** [comment]
We intentionally try to keep the hello world example simple and not raise/handle exceptions and errors. Probably a separate example to handle various error cases like server could not start or handler raises exception etc. would be better.

**Comment by gnossen** [comment]
@lidizheng Brought up a good point related to this. While users feel it's [their right](https://github.com/grpc/grpc/issues/17135) to set up their own logging configuration, we as library maintainers feel it is [our *responsibility*](https://github.com/grpc/grpc/issues/17321) to ensure that the default logging configuration is a sane one. To achieve a happy medium, perhaps we could wait until server start, detect if no handlers have been set up, and if not, install loggers to `stdout` at `info` level?

One caveat I would add is that I have not seen any other libraries implement this sort of behavior.

**Comment by lidizheng** [comment]
Fixed by #17322
